staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -73,7 +73,9 @@ type ListMeta struct {
 
 // These are internal finalizer values for Kubernetes-like APIs, must be qualified name unless defined here
 const (
-	FinalizerOrphan string = "orphan"
+	FinalizerOrphan           string = "orphan"
+	FinalizerOrphanDependents string = "orphan"
+	FinalizerDeleteDependents string = "foregroundDeletion"
```

Comment:
Why not qualify this?
IMO, the non qualified name exception was only for existing qualifiers. We should qualify new finalizers.